.eslintrc.js

```diff
@@ -49,8 +47,17 @@ module.exports = {
     "@typescript-eslint/no-use-before-define": 0,
     "@typescript-eslint/no-unused-vars": 0,
     "@typescript-eslint/no-var-requires": 0,
+    "react/display-name": 0,
+    "react/prop-types": 0,
     "react-hooks/rules-of-hooks": "error",
     "react-hooks/exhaustive-deps": "warn",
+    "sort-imports": [
+      "warn",
+      {
+        // Unfortunately there's no autofixer for this
+        ignoreDeclarationSort: true,
```

Comment:
In which case we lean on the autofixer to sort named export keys on individual imports, but don't worry about alpha for the full import list as thats a real pain.